src/skdh/completeness/complete.py

```diff
@@ -0,0 +1,127 @@
+import os
+import pickle
+import numpy as np
+from skdh.base import BaseProcess
+from skdh.completeness.utils import load_subject_data, compute_completeness_master, compute_summary_metrics, check_hyperparameters
+from skdh.completeness.visualizations import visualize_overview_plot, plot_completeness, plot_data_gaps, plot_timescale_completeness
+
+
+class completeness_pipe(BaseProcess):
```

Comment:
class names should be CamelCase, suggest changing name to something like "AssessDataCompleteness" as well

Reply:
renamed to AssessCompleteness